client/client.go

```diff
@@ -375,18 +375,20 @@ func loadServerTrust(ctx *openssl.Ctx, conf *Config) (*openssl.Ctx, error) {
 			err.Error(),
 		)
 	}
-	// Load the server certificate into the OpenSSL context
-	err = ctx.LoadVerifyLocations(conf.ServerCert, "")
-	if err != nil {
-		if strings.Contains(err.Error(), "No such file or directory") {
-			log.Warnf(errMissingServerCertF, conf.ServerCert)
-		} else {
-			log.Errorf("Failed to Load the Server certificate. Err %s", err.Error())
-		}
-		// If no system certificates, nor a server certificate is found,
-		// warn the user, as this is a pretty common error.
-		if sysCertsFound == 0 {
-			log.Error(errMissingCerts)
+	if conf.ServerCert != "" {
+		// Load the server certificate into the OpenSSL context
+		err = ctx.LoadVerifyLocations(conf.ServerCert, "")
+		if err != nil {
+			if strings.Contains(err.Error(), "No such file or directory") {
+				log.Warnf(errMissingServerCertF, conf.ServerCert)
+			} else {
+				log.Errorf("Failed to Load the Server certificate. Err %s", err.Error())
+			}
+			// If no system certificates, nor a server certificate is found,
+			// warn the user, as this is a pretty common error.
+			if sysCertsFound == 0 {
+				log.Error(errMissingCerts)
+			}
```

Comment:
I am fine with dropping this error, but it seems we are doing that one on purpose, perhaps we could live with a warning when conf.ServerCert is empty and we have no certificates in default path? that is, maybe we should keep it at warning in a separate if:

```go
if conf.ServerCert == "" && sysCertsFound == 0  {
        log.Warn(errMissingCerts)
}
```